exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -454,3 +454,21 @@ func (b *Bitfinex) GetOrderHistory(getOrdersRequest *exchange.GetOrdersRequest)
 
 	return orders, nil
 }
+
+// SubscribeToWebsocketChannels appends to ChannelsToSubscribe
+// which lets websocket.manageSubscriptions handle subscribing
+func (b *Bitfinex) SubscribeToWebsocketChannels(channels []exchange.WebsocketChannelSubscription) error {
+	for i := range channels {
```

Comment:
This logic should be moved to the websocket package instead of the individual exchanges to reduce the requirement of for looping over the supplied channels for every exchange wrapper